src/searchbar.vue

```diff
@@ -0,0 +1,57 @@
+<script>
+import Typeahead from './Typeahead.vue';
```

Comment:
- Lowercase for imported variable:

  ```js
  import typeahead from './Typeahead.vue';
  ```

- Uppercase for the name of this file: **Searchbar.vue**

Reply:
but all other files in that folder is lowercase.

Reply:
Since when? https://github.com/MarkBind/vue-strap/tree/master/src

Reply:
oops, I was looking at the docs files...